freqtrade/rpc/api_server/api_schemas.py

```diff
@@ -456,5 +456,5 @@ class SysInfo(BaseModel):
 
 
 class Health(BaseModel):
-    last_process: datetime
-    last_process_ts: int
+    last_process: Optional[datetime]
+    last_process_ts: Optional[int]
```

Comment:
@froggleston  i know you use the /health endpoint in frogtrade9000 ... is this going to cause problems? 
This would happen anywhere between start of the bot and the first iteration .... so only for a quite limited amount of time.

Reply:
No, I only use sysinfo in frogtrade9000 so no problems I can see.